core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/TableSink.java

```diff
@@ -74,6 +75,12 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
   }
 
   @Override
+  protected boolean shouldSkipCreateAtConfigure() {
+    return tableSinkConfig.containsMacro(Properties.Table.PROPERTY_SCHEMA) ||
```

Comment:
if we create them at configure time, schema will be unavailable and they will not be explorable